apps/condo/domains/meter/schema/_internalDeleteMeterAndMeterReadingsService.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createservice meter._internalDeleteMeterAndMeterReadingsService --type mutations`
+ */
+
+const { isEmpty } = require('lodash')
+
+const { GQLCustomSchema, find, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/meter/access/_internalDeleteMeterAndMeterReadingsService')
+
+const { Meter } = require('../utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+
+const _internalDeleteMeterAndMeterReadingsService = new GQLCustomSchema('_internalDeleteMeterAndMeterReadingsService', {
+    schemaDoc: 'Mutation to delete meters and meter readings for property',
+    types: [
+        {
+            access: true,
+            // TODO(codegen): write DeleteMeterAndMeterReadingsService input !
+            type: 'input _internalDeleteMeterAndMeterReadingsInput { dv: Int!, sender: JSON!, propertyIds: [String]! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.can_internalDeleteMeterAndMeterReadings,
+            schema: '_internalDeleteMeterAndMeterReadings(data: _internalDeleteMeterAndMeterReadingsInput!): [MeterReading]',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { dv, sender, propertyIds } = data
+
+                const findMeters = async () => {
```

Comment:
you do not use findMeters function more then once 
better to use direct find here 
```
const meters = await find('Meter', {
                        deletedAt: null,
                        property: {
                            id_in: propertyIds,
                        },
                    })
```